bom/application/pom.xml

```diff
@@ -46,7 +46,7 @@
         <microprofile-jwt.version>2.1</microprofile-jwt.version>
         <microprofile-lra.version>2.0.1</microprofile-lra.version>
         <microprofile-openapi.version>4.0.2</microprofile-openapi.version>
-        <smallrye-common.version>2.13.5</smallrye-common.version>
+        <smallrye-common.version>2.13.6</smallrye-common.version>
```

Comment:
You also need to bump https://github.com/quarkusio/quarkus/blob/main/independent-projects/junit5-virtual-threads/pom.xml#L45